README.md

```diff
@@ -1,70 +1,55 @@
-# mbed-os-example-psa
+# PSA Mbed OS example
```

Comment:
Is the name appropriate, though it was Mbed OS - PSA example.

Reply:
This is following convention of other core examples.